openedx_tagging/core/tagging/api.py

```diff
@@ -26,7 +26,6 @@ def create_taxonomy(
     name: str,
     description: str | None = None,
     enabled=True,
-    required=False,
     allow_multiple=False,
```

Comment:
Could you change this to match the new database default?

```suggestion
    allow_multiple=True,
```